markdown/extensions/toc.py

```diff
@@ -207,6 +207,7 @@ def build_toc_div(self, toc_list):
 
         def build_etree_ul(toc_list, parent):
             ul = etree.SubElement(parent, "ul")
+            ul.attrib["class"] = "tocstyle-%d" % toc_list[0]['level']
```

Comment:
I'm not crazy about the name of the class. Why do we need "style" in the class name. Yes, you might use the class as a styling hook, but you could also use it from JavaScript or for some other purpose. A more generic name seems better to me. Sorry, I don't have any suggestions.

Reply:
My consideration was not enough. How about `toclist-*`?

Reply:
I suppose that works. Why not just `toc` on the top level only? The nested levels seem redundant to me.

Reply:
Thank you for review. Also most of the cases are top level `tos` is fine. Sometime wants to adjust font-style of nested place.